src/components/TokenSafety/TokenSafetyIcon.tsx

```diff
@@ -0,0 +1,39 @@
+import { Warning, WARNING_LEVEL } from 'constants/tokenSafety'
+import { useTokenWarningColor } from 'hooks/useTokenWarningColor'
+import { AlertOctagon, AlertTriangle } from 'react-feather'
+import styled from 'styled-components/macro'
+import { Color } from 'theme/styled'
+
+import { ReactComponent as Verified } from './verified.svg'
```

Comment:
verified should be in assets, not in the component folder

Reply:
@kayleegeorge can you move